__tests__/app.test.js

```diff
@@ -264,6 +264,36 @@ describe('PATCH /api/articles/:article_id', () => {
     })
 })
 
+describe('DELETE /api/comments/:comment_id', () => {
+    test('DELETE 204 deletes a given comment', () =>{
+        return request(app)
+        .delete('/api/comments/3')
+        .expect(204)
+        .then(() => {
+            return db.query(`SELECT * FROM COMMENTS`)            
+        })
+        .then(({rows}) => {
+            expect(rows).toHaveLength(17)
+        })
```

Comment:
⭐ ⭐ ⭐ I love that you were cautious and made this extra query to confirm the success of the delete